ipn/message.go

```diff
@@ -31,9 +31,11 @@ type FakeExpireAfterArgs struct {
 	Duration time.Duration
 }
 
-// A command message sent to the server. Exactly one of these must be non-nil.
+// Command is a command message sent by a frontend to a backend.
```

Comment:
clarify that it's JSON-encoded, since I haven't done a pass over all these to add explicit JSON struct tags.